---
layout: page
title: Glossary of ancient metagenomics acronyms
sidebar_link: false
---

I hate acronyms. Here is a list of them. DO NOT USE THEM. Please write them out in full so my acronym-dyslexic brain can cope. Thanks. 

## Programs
| Acronym   | Actual name          | Short description     | Uniform Resource Locator (URL)       |
|----------:|:--------------------:|:---------------------:|:------------------------------------:|
|metaDMG   | Metagenomics DaMaGe  | Estimation, quantification and visualization of postmortem damage of single reads, single genomes and metagenomic environmental DNA | [GitHub](https://github.com/metaDMG-dev/metaDMG-cpp) |
|MALT      | Megan ALignment Tool | High-throughput Metagenomic BLAST-like alignment tool | [Website](https://software-ab.cs.uni-tuebingen.de/download/malt/welcome.html) |
|HOPS      | Heuristic Operations for Pathogen Screening | Tool for the authentication of ancient DNA | [GitHub](https://github.com/rhuebler/HOPS) |
|PyDamage  | Python Damage        | Estimates DNA damage for de novo-assembled contigs | [GitHub](https://github.com/maxibor/pydamage) |
|
|Fastp
|FastQC
|MultiQC
|Metahit
|MEGAN
|bwa aln
bwa mem
|bowtie
|filterBAM/bam-filter
|AMDirt
BEAST


## Pipelines
nf-core/eager
nf-core/mag
HOPS

## Databases
GenBank
BUSCO
GTDD
GUNC

## File formats
.fastq
.fasta
.bam/.sam/.cram
.bai

## Misc
SPAAM
MAG/SMAG
USER or UDG
IGV
sedaDNA
aeDNA
eDNA
SNP
ANI
